shogun-lib/src/main/java/de/terrestris/shogun/lib/service/CacheService.java

```diff
@@ -43,4 +47,23 @@ public void evictCache() throws Exception {
 
         cache.evictAllRegions();
     }
+
+    public void evictCacheRegions(String... region) throws Exception {
+        if (region == null) {
+            return;
+        }
+        if (entityManager == null) {
+            throw new Exception("Could not get the entity manager.");
+        } else {
```

Comment:
`else` is not necessarily needed.

Reply:
indeed! removed in e2e58ba45085a61b7588768bf83814757a683e13